### PR TITLE
fix: validate README.md

### PR DESCRIPTION
**Pull request by dgarus**
Part of https://github.com/artipie/artipie/issues/1031

- added telegram icon and lint to contributing guide
- added license update action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +30,9 @@ This is the dependency you need:
 
 Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/vertx-server)
 for more technical details.
-If you have any question or suggestions, do not hesitate to create an issue or contact us in
-[Telegram](https://t.me/artipie). Artipie roadmap is available [here](https://github.com/orgs/artipie/projects/3).]
+If you have any question or suggestions, do not hesitate to [create an issue](https://github.com/artipie/http-client/issues/new) or contact us in
```
Comment (Swizbiz) [inline README.md]: change http-client to vertx-server in link https://github.com/artipie/http-client/issues/new

**Review by Swizbiz — CHANGES_REQUESTED** [review]
please check my comment

File: README.md (hunk 1 above)
Comment (dgarus) [reply README.md]: done

**Review by Swizbiz — APPROVED** [review]
thanks!

**Pull request merged**